dipy/workflows/segment.py

```diff
@@ -358,14 +359,27 @@ def run(self, input_files, save_masked=False, out_dir='',
             Name of the mask volume to be saved.
         out_masked : string, optional
             Name of the masked volume to be saved.
+
+        References
+        ----------
+        ..  [Park2022] Park, J.S., Fadnavis, S., & Garyfallidis, E. (2022).
+        EVAC+: Multi-scale V-net with Deep Feature
+        CRF Layers for Brain Extraction.  
+
         """
+        if os.path.exists(out_mask) or os.path.exists(out_masked):
```

Comment:
it should not be done here. it should be done in the for loop with `mask_out_path` and `masked_out_path`